packages/data/addon/mirage/factories/metric.ts

```diff
@@ -1,13 +1,14 @@
 /**
- * Copyright 2020, Yahoo Holdings Inc.
+ * Copyright 2021, Yahoo Holdings Inc.
  * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
  */
-import { Factory } from 'ember-cli-mirage';
+import { Factory } from 'miragejs';
 
 export default Factory.extend({
-  index: i => i,
+  index: (i: number) => i,
 
-  id() {
+  id(): string {
+    //@ts-ignore
```

Comment:
What's the issue here? Accessing properties on `this`?

Reply:
I couldn't figure out how to use the `'miragejs'` types to get the definition of `'table'` (we only define the factory and I was struggling to figure out how to extract the instance definition) so I just ignored it for now since I didn't think it was worth investigating